marklogic-data-hub-central/ui/src/components/load/create-edit-load/create-edit-load.test.tsx

```diff
@@ -95,7 +95,7 @@ describe('New/edit load step configuration', () => {
   });
 
   test('Verify name is required for form submission and error messaging appears', async () => {
```

Comment:
@AkshayBajajML have you already tried adding a unit test for the tooltip in this form as well?

Reply:
Thank you for the update @briantang. Mike has already submitted separate ticket to test this functionality for whole basic/Advanced setting modals(https://project.marklogic.com/jira/browse/DHFPROD-6197)